### PR TITLE
Add rails 5.0.0.beta1 support

### PR DESCRIPTION
**Pull request by daande**
Add rails 5.0.0.beta1 support

### CODE REVIEW

**Comment by jrgifford** [comment]
Pending CI, looks good to me!

**Comment by dnrce** [comment]
Thanks!

Rails 5 needs to be added to the Travis build matrix in .travis.yml in order for it to actually be "supported". Can you please make that addition? Then we'll see how the build goes. :-)

Please also squash your commits and improve the title -- "Add rails 5.0.0.beta1 support" is much more explanatory than "Update yaml_db.gemspec".

**Comment by dnrce** [comment]
Oh, also the README and the CHANGELOG should be updated with this change.

File: yaml_db.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 1.8.7"
 
-  s.add_runtime_dependency "rails", ">= 3.0", "< 4.3"
+  s.add_runtime_dependency "rails", ">= 3.0", "<= 5.0.0.beta1"
```
Comment (dnrce) [inline yaml_db.gemspec]: Will it work to do `< 5.1`?

Comment (daande) [inline yaml_db.gemspec]: I think < 5.1 would work. I will try it tomorrow

**Comment by dnrce** [comment]
FYI here is what we did when Rails 4.2.0 came out: https://github.com/yamldb/yaml_db/commit/b444524f89fb88fa3e8cd33d51261460f735a763

**Comment by dnrce** [comment]
...and back when it was in beta: f56f074e2e668926fd4a2d14475ac362545b2606

**Comment by daande** [comment]
I am having trouble with my access pushing from a local repo to my master fork https://github.com/daande/yaml_db

**Comment by dnrce** [comment]
What trouble?

**Comment by dnrce** [comment]
You'll want to rebase this over the current master in order to pick up some CI fixes from #89.

**Comment by dnrce** [comment]
@daande, any update on this?

**Pull request closed without merging**